docs/source/WritingTests.rst

```diff
@@ -58,6 +58,86 @@ Note that the test class provides you with a number of convenience attributes:
   you can find that more information at :doc:`Mux`.
 * And many more (see :mod:`avocado.core.test.Test`)
 
+Test statuses
+=============
+
+Avocado supports the most common exit statuses:
+
+* ``PASS`` - test passed, there were no untreated exceptions
+* ``WARN`` - the test passed, but there were some warnings in the main
+  avocado log. The purpose of this is to say that the item under testing
+  behaved correctly, but there were some (probably unrelated) things which
+  might or might not require attention. An example could be ``soft lockup``
+  present in the ``dmesg`` output. It's not related to the test results
+  and unless there are failures in the test it means the feature probably
+  works as expected, but there were certain condition which might be nice
+  to review. (some result plugins does not support this and report
+  ``PASS`` instead)
+* ``SKIP`` - the test's pre-requisites were not satisfied and the test
+  body was not executed (nor the ``tearDown``)
+* ``FAIL`` - the object under testing failed in expected manner. There is
```

Comment:
Suggestion:  "test did not result in the expected outcome.  A failure points at a (possible) bug in the tested subject, and not in the test itself.  When the test (and its) execution breaks, an ``ERROR`` and not a ``FAIL`` is reported."